.travis.yml

```diff
@@ -15,16 +15,22 @@ before_install:
   - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
 
-script:
-  - make test-coverage codecov
+stages:
+  - name: release to github
+    if: tag IS present
 
 jobs:
   include:
     - stage: "Tests"
       name: "Unit Tests"
       script: make test-coverage codecov
     - script: make test-sdk
-      name: "Integration Tests"
+      name: "SDK Integration Tests Linux"
+    - script: make test-sdk
+      name: "SDK Integration Tests macOS"
+      os: osx
+      osx_image: xcode9.4
+      before_install: echo "skipping before_install for macOS"
```

Comment:
Here and below, just wondering, why not using `skip` or `ignore` [keyword](https://docs.travis-ci.com/user/build-stages/#Build-Stages-and-Deployments) ?

Reply:
For no special reason, I just copied it from [gitbase travis.yml](https://github.com/src-d/gitbase/blob/master/.travis.yml) and I was not aware of the skip keyword. I'll replace it before merging.